commons/ava_networks/high_level_gecko_client.go

```diff
@@ -274,8 +275,18 @@ func (highLevelGeckoClient HighLevelGeckoClient) TransferAvaPChainToXChain(
 	}
 	// XChain API only accepts the XChain address with the xchain prefix.
 	txnId, err := client.XChainApi().ImportAVA(xchainAddress, username, password)
-	if err != nil {
-		return "", stacktrace.Propagate(err, "Failed import AVA to xchainAddress %s", xchainAddress)
+	for err != nil {
+		/*
+			HACK HACK HACK because the PChain does not have a way to verify transaction acceptence yet,
+			we retry based on the contents of the error message from the XChain call if the pchain transaction
+			has not yet reached consensus
+		*/
+		// TODO TODO TODO When the PChain transaction status endpoint is deployed, use that to wait fro transaction acceptance
+		if strings.Contains(err.Error(), NO_IMPORT_INPUTS_ERROR_STR) {
+			txnId, err = client.XChainApi().ImportAVA(xchainAddress, username, password)
```

Comment:
This probably needs to be in a "retry 5 times, with 1s in between" loop

Reply:
@mieubrisse  right now this retries indefinitely until the test times out, why limit it to 5 retries?